latex2sympy.py

```diff
@@ -296,7 +296,7 @@ def convert_postfix_list(arr, i=0):
         raise Exception("Index out of bounds")
 
     res = convert_postfix(arr[i])
-    if isinstance(res, sympy.Expr) or isinstance(res, sympy.Matrix):
+    if isinstance(res, sympy.Expr) or isinstance(res, sympy.Matrix) or res is sympy.S.EmptySet:
```

Comment:
Can this actually happen? Just curious since infty and pi aren't checked here but are symbols too.

Reply:
@tragessere yeah the one emptyset test was failing here. I guess it isn't technically an expression

Reply:
or at least not a sympy.Expr